images/build/Dockerfiles/ltsVersions.buster.Dockerfile

```diff
@@ -198,9 +198,6 @@ RUN --mount=type=secret,id=oryx_sdk_storage_account_access_token \
     && echo "DEBIAN|${DEBIAN_FLAVOR}" | tr '[a-z]' '[A-Z]' > /opt/oryx/.ostype \
     # as per solution 2 https://stackoverflow.com/questions/65921037/nuget-restore-stopped-working-inside-docker-container
     && ${imagesDir}/retry.sh "curl -o /usr/local/share/ca-certificates/verisign.crt -SsL https://crt.sh/?d=1039083" \
-    && update-ca-certificates \
-    && echo "value of DEBIAN_FLAVOR is ${DEBIAN_FLAVOR}" \
```

Comment:
Same comment here -- line was accidentally removed

Reply:
thanks for finding it. fixed